app/views/shared/_accounts_table.html.haml

```diff
@@ -7,16 +7,10 @@
     - @accounts.each do |account|
       %tr
         %td
-          - case
-          - when !current_ability.can?(:manage, account)
-            = account
-
-          - when current_facility.cross_facility?
+          - if current_ability.can?(:manage, account)
             = link_to account, facility_account_path(current_facility, account)
-
           - else
-            = link_to account,
```

Comment:
You're right, with this change I don't see any way for a user to navigate  to `facility_account_orders_path`.

What do you think about adding an "orders" tab to `app/views/admin/shared/_tabnav_payment_method.html.haml`? This is the horizontal nav when you're at `/facilities/:facility_id/accounts/:account_id`.